### DISCUSSION TITLE
Ribbon menu

### DISCUSSION THREAD

**Discussion by eduardoagr** [CommunityToolkit/Maui#1842] [New Feature Discussions]
I created a Ribbon menu, I know that this control only makes sense for desktop, but if anyone is interested, I can throw it in the MCT.

![image](https://github.com/CommunityToolkit/Maui/assets/6854010/2a418cb0-1425-49da-ae46-a8a1383242fe)

**Comment by CliffAgius** [comment]
Closed as per the Community Stand-up discussion here: https://youtu.be/N9wMcBP4jtg?t=2889 where it was agreed that New Features need to get above 7 upvotes within 6 months to progress to being reviewed by the maintainers. 

If you feel this is being closed in error please let us know and we can re-open.

**Comment by VladislavAntonyuk** [comment]
@eduardoagr is it open source?

**Reply by eduardoagr** [reply]
Yes, but I wanted to include it in the mct, but I have to look for it again
